chia/cmds/cmd_classes.py

```diff
@@ -48,6 +62,29 @@ def option(*param_decls: str, **kwargs: Any) -> Any:
     )
 
 
+@final
+@dataclasses.dataclass
+class ChiaCliContext:
+    context_dict_key: ClassVar[str] = "_chia_cli_context"
+
+    root_path: pathlib.Path = DEFAULT_ROOT_PATH
+    expected_prefix: Optional[str] = None
+    rpc_port: Optional[int] = None
+    keys_fingerprint: Optional[int] = None
+    keys_filename: Optional[str] = None
+    expected_currency_prefix: Optional[str] = None
```

Comment:
Nittest of nits but can we change this to address prefix?  Not everything being described will be a "currency" (think NFTs, DIDs, etc.)

Also, not sure (yet, maybe) what the difference is between this and expected_prefix.

Reply:
```suggestion
    expected_address_prefix: Optional[str] = None
```